packages/fxa-auth-server/lib/senders/emails/EmailStory/index.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import RenderedEmail, { RenderedEmailProps } from '../RenderedEmail';
+
+export default {
+  title: 'Emails',
+} as Meta;
+
+const Template: Story<RenderedEmailProps> = (args) => (
+  <RenderedEmail {...args} />
+);
+
+export const CadReminder = Template.bind({});
+CadReminder.args = {
+  template: 'cadReminder',
+  variables: {
+    buttonText: 'Sync device',
+    onDesktopOrTabletDevice: true,
+    anotherDeviceURL:
+      'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-first&utm_content=fx-connect-device',
```

Comment:
I think we'll need to conditionally attach the base URL so it's not hard-coded to localhost